### ISSUE TITLE
Error when using only variable placeholder in the texts for internationalization

### ISSUE THREAD

**Issue by ramonamorea** [WordPress-Coding-Standards/WordPress-Coding-Standards#1646]
## Bug Description
When I use only the variable placeholder in the texts for internationalization, then an error is shown.

## Minimal Code Snippet
```php
    printf(
        /* translators: %s: A message shown to admin users in the admin dashboard */
        esc_html__( **'%s'**, 'my-plugin' ),
        $message
    );
```

So I only use the '%s' placeholder, not  'Attention here to this message: %s'.

## Error Code
Then it shows this PHP Code Sniffer error: "Strings should have translatable content"
But it shouldn't show this error, using only the placeholder should be just fine.

## Environment
| PHP version             | 7.1.11
| PHP_CodeSniffer version | 3.4.0
| WPCS version            | 2.0.0
| WPCS install type       | Composer global
| IDE (if relevant)       | Eclipse Neon.3 Release (4.6.3)

**Comment by jrfnl** [comment]
@ramonamorea Sorry, but no, this is not a false positive as there is no text string to translate.

Calling a translation function without translatable text is unnecessary overhead and useless, quite apart from wasting the time of translators who can't do anything with "string" like that.

Your code - based on the above code sample - should probably be changed to something along the lines of:
```php
    echo esc_html( $message );
    // or
    echo wp_kses_post( $message );
```

Closing as duplicate of #1537.
